### PR TITLE
Support Modify ViewDef binlog

### PR DESCRIPTION
**Pull request by xiaoming12306**
Support Modify ViewDef binlog

### CODE REVIEW

File: pkg/rpc/kitex_gen/frontendservice/FrontendService.go [hunk 1]
```diff
@@ -629,6 +629,7 @@ const (
 	TBinlogType_TRUNCATE_TABLE                   TBinlogType = 13
 	TBinlogType_RENAME_TABLE                     TBinlogType = 14
 	TBinlogType_RENAME_COLUMN                    TBinlogType = 15
+	TBinlogType_MODIFY_VIEW_DEF                  TBinlogType = 16
```
Comment (smallx) [inline pkg/rpc/kitex_gen/frontendservice/FrontendService.go]: 16 -> 17

File: pkg/rpc/thrift/FrontendService.thrift [hunk 2]
```diff
@@ -1184,6 +1184,7 @@ enum TBinlogType {
   TRUNCATE_TABLE = 13,
   RENAME_TABLE = 14,
   RENAME_COLUMN = 15,
+  MODIFY_VIEW_DEF = 16,
```
Comment (smallx) [inline pkg/rpc/thrift/FrontendService.thrift]: 16 -> 17

File: pkg/rpc/kitex_gen/frontendservice/FrontendService.go (hunk 1 above)
Comment (xiaoming12306) [reply pkg/rpc/kitex_gen/frontendservice/FrontendService.go]: Thank you for the rewives. I have already addressed this issue

**Comment by w41ter** [comment]
@xiaoming12306 hi, please resolve the conflicts, I will merge it once all conflicts are resolved.

**Comment by xiaoming12306** [comment]
@w41ter Thank you for your comment. I have made the necessary changes

**Comment by xiaoming12306** [comment]
> @xiaoming12306 hi, please resolve the conflicts, I will merge it once all conflicts are resolved.

Thank you for your comment. I have made the necessary changes

File: pkg/ccr/job.go [hunk 3]
```diff
@@ -2182,6 +2182,32 @@ func (j *Job) handleBarrier(binlog *festruct.TBinlog) error {
...
+    	viewName, err := j.destMeta.GetTableNameById(tableId)
+    	if err != nil {
+    		return err
+    	} else if viewName == "" {
+    		return xerror.Errorf(xerror.Normal, "tableId %d not found in destMeta", tableId)
+    	}
+
+    	err = j.IDest.AlterViewDef(viewName, alterView)
+    	return err
+}
```
Comment (w41ter) [inline pkg/ccr/job.go]: Format it

File: pkg/ccr/record/alter_view.go [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+func NewAlterViewFromJson(data string) (*AlterView, error) {
+	var alterView AlterView
+	err := json.Unmarshal([]byte(data), &alterView)
+	if err != nil {
+		return nil, fmt.Errorf("unmarshal alter view error: %v", err)
+	}
+
+	if alterView.TableId == 0 {
+    	return nil, fmt.Errorf("table id not found")
```
Comment (w41ter) [inline pkg/ccr/record/alter_view.go]: format it

File: pkg/ccr/job.go (hunk 3 above)
Comment (xiaoming12306) [reply pkg/ccr/job.go]: Thank you for your comment. I have made the necessary changes

File: pkg/ccr/record/alter_view.go (hunk 4 above)
Comment (xiaoming12306) [reply pkg/ccr/record/alter_view.go]: Thank you for your comment. I have made the necessary changes

**Review by w41ter — APPROVED** [review]

**Pull request merged**